packages/edit-navigation/src/components/layout/index.js

```diff
@@ -172,7 +172,6 @@ export default function Layout( { blockEditorSettings } ) {
 														isPending={
 															! hasLoadedMenus
 														}
-														blocks={ blocks }
```

Comment:
I noticed this while updating the example code. The `Editor` doesn't use the `blocks` prop since #28675.